perfkitbenchmarker/gcp/gce_network.py

```diff
@@ -30,6 +33,56 @@
 FLAGS = flags.FLAGS
 
 
+class GceFirewallRule(resource.BaseResource):
+  """An object representing a GCE Firewall Rule."""
+
+  def __init__(self, name, project, port):
+    super(GceFirewallRule, self).__init__()
+    self.name = name
+    self.project = project
+    self.port = port
+
+  def __eq__(self, other):
+    """Defines equality to make comparison easy."""
+    return self.__dict__ == other.__dict__
```

Comment:
Would you consider two firewall rules to be non-equal if the only differences are in `created` or `deleted`?